Gemfile

```diff
@@ -24,7 +24,7 @@ group :local_development do
   gem 'guard-bundler', require: false
   gem 'guard-preek', require: false
   gem 'guard-rubocop', require: false
-  gem 'guard-reek', github: 'pericles/guard-reek', require: false
+  gem 'guard-reek', git: 'https://github.com/pericles/guard-reek', require: false
```

Comment:
Compatibility issues with proxies and such which don't like OPTION request. It's just easier to maintain and live with, in my opinion. But you can drop it.